src/main/java/fr/esgi/al5/tayarim/services/LogementService.java

```diff
@@ -326,4 +328,24 @@ public List<TypeLogementDto> getAllTypeLogement() {
         typeLogementRepository.findAll()
     );
   }
+
+  public Logement updateLogement(LogementUpdateDto logementUpdateDto, Logement logement) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.coding.OverloadMethodsDeclarationOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>All overloaded methods should be placed next to each other. Previous overloaded method located at line '154'.